### PR TITLE
KAS-4683: Cabinet submissions

### PR DESCRIPTION
**Pull request by sergiofenoll**
https://kanselarij.atlassian.net/browse/KAS-4683

Adds the necessary types and config to enable cabinets to submit data to Kaleidos.

:warning: This will be hard to test without the follow up frontend tickets (where the actual UI for creating/viewing is added).

### CODE REVIEW

File: config/new-authorization/config.lisp [hunk 1]
```diff
@@ -528,6 +529,39 @@
...
        :for-allowed-group "parliament-flow-write")
 
+;; Submissions data
+
+(define-graph system/submissions ("http://mu.semte.ch/graphs/system/submissions")
+  ("subm:Indiening" -> _)
+  ("subm:StatusVeranderingsActiviteit" -> _)
+  ("subm:Serie" -> _)
+  ("subm:VoorlopigStuk" -> _)
+  ("subm:VoorloopigBestand" -> _))
```
Comment (tdn) [inline config/new-authorization/config.lisp]: `subm:VoorloopigBestand` -> `subm:VoorlopigBestand`

File: config/resources/submissions-domain.lisp [hunk 2]
```diff
@@ -0,0 +1,110 @@
+(define-resource submission ()
+  :class (s-prefix "subm:Indiening")
+  :properties `((:short-title         :string ,(s-prefix "dct:alternative"))
+                (:title               :string ,(s-prefix "dct:title"))
```
Comment (tdn) [inline config/resources/submissions-domain.lisp]: just as a remark: this is set to make a new `decisionmakingflow`

File: config/resources/submissions-domain.lisp [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+                (:confidential        :boolean   ,(s-prefix "ext:vertrouwelijk"))
+                (:subcase-name        :string ,(s-prefix "ext:procedurestapNaam"))
```
Comment (tdn) [inline config/resources/submissions-domain.lisp]: subcase-name is not necessary

File: config/migrations/20240604133010-submission-status.ttl [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+@prefix thConceptScheme: <http://themis.vlaanderen.be/id/concept-scheme/> .
+
+thConceptScheme:ebfe253c-0537-11ee-bb35-ee395168dcf7 a skos:ConceptScheme ;
+  mu:uuid "9ee8c485-e750-4982-a2b7-3f17d390331d" ;
+  skos:prefLabel "Indiening statussen" .
+
+<http://themis.vlaanderen.be/id/indieningstatus/b8a0cec4-60d5-431e-b27f-9696bde22552> a skos:Concept ;
+    mu:uuid "b8a0cec4-60d5-431e-b27f-9696bde22552" ;
+    skos:inScheme thConceptScheme:9ee8c485-e750-4982-a2b7-3f17d390331d ;
+    skos:prefLabel "Ingediend" .
```
Comment (tdn) [inline config/migrations/20240604133010-submission-status.ttl:14]: @ValenberghsSven suggests to add `schema:position` here

File: config/resources/submissions-domain.lisp [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+  :has-one `((decisionmaking-flow     :via ,(s-prefix "subm:ingediendVoor")
+                                      :as "decisionmaking-flow")
+             (subcase                 :via ,(s-prefix "subm:ingediendVoorProcedurestap")
+                                      :as "subcase")
+             (subcase-type            :via ,(s-prefix "dct:type")
+                                      :as "type")
+             (concept                 :via ,(s-prefix "ext:agendapuntType")
+                                      :as "agenda-item-type")
+             (concept                 :via ,(s-prefix "adms:status")
+                                      :as "status")
```
Comment (tdn) [inline config/resources/submissions-domain.lisp]: add comment: this is the submissions status (concept-scheme `thConceptScheme:ebfe253c-0537-11ee-bb35-ee395168dcf7`)

File: docker-compose.yml [hunk 6]
```diff
@@ -497,3 +497,13 @@ services:
       MU_APPLICATION_FILE_STORAGE_PATH: 'stamped-pdfs'
     restart: always
     logging: *default-logging
+  draft-file:
+    image: kanselarij/draft-file-service:0.0.1
+    environment:
+      FILE_RESOURCE_BASE: "http://themis.vlaanderen.be/id/bestand/"
```
Comment (tdn) [inline docker-compose.yml]: change to http://themis.vlaanderen.be/id/voorlopig-bestand/

**Review by tdn — CHANGES_REQUESTED** [review]
Some minimal changes/cleanup

**Review by ValenberghsSven — APPROVED** [review]
tested

**Pull request merged**